pom.xml

```diff
@@ -1287,7 +1287,7 @@
         <carbon.kernel.version>4.5.3</carbon.kernel.version>
         <carbon.kernel.imp.pkg.version>[4.5.0,5.0.0)</carbon.kernel.imp.pkg.version>
 
-	<synapse.version>2.1.7-wso2v156</synapse.version>
+	<synapse.version>2.1.7-wso2v157</synapse.version>
```

Comment:
this has to be 158 ? Also please format this.

Reply:
just wanted to chet with 157 since the release had completed, what's the formatting issue?